doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1319,16 +1327,16 @@ tactic language. Tactic notations obey the following syntax:
 
 .. cmdv:: Local Tactic Notation
 
-   Tactic notations do not survive the end of sections. They survive
-   modules unless the command Local Tactic Notation is used instead of
-   Tactic Notation.
+   Tactic notations disappear when a section is closed. They survive when
+   a module is closed unless the command ``Local Tactic Notation`` is used instead
+   of :cmd:`Tactic Notation`.
 
 .. rubric:: Footnotes
 
 .. [#and_or_levels] which are the levels effectively chosen in the current
    implementation of Coq
 
-.. [#no_associativity] Coq accepts notations declared as ``no associative`` but the parser on
+.. [#no_associativity] Coq accepts notations declared as ``non-associative`` but the parser on
```

Comment:
It was in fact ``no associativity`` and has been fixed in another PR (this explains the conflict).